server/api.js

```diff
@@ -42,4 +42,25 @@ router.delete("/videos/:id", async (req, res) => {
 	}
 });
 
+router.put("/videos/:id/rating", async (req, res) => {
+	const videoId = req.params.id;
+	const { rating } = req.body;
+
+	try {
+		const result = await db.query(`SELECT * FROM videos WHERE id=${videoId}`);
```

Comment:
As mentioned last time, this way of  composing SQL isn't very secure - talk to Zeliha about the way she does it which is correct

Reply:
done it!